include/swift/AST/Attr.def

```diff
@@ -560,6 +560,11 @@ SIMPLE_DECL_ATTR(noDerivative, NoDerivative,
   ABIBreakingToAdd | ABIBreakingToRemove | APIBreakingToAdd | APIBreakingToRemove,
   100)
 
+DECL_ATTR(requiresSuper, RequiresSuper,
+  OnFunc |
+  ABIStableToAdd | ABIStableToRemove | APIStableToAdd | APIStableToRemove,
```

Comment:
This is most certainly *not* API stable to add in a pure Swift context.

Reply:
You're right, I copy pasted it from the decl above but forgot to swap it with `APIBreakingToAdd` 🤦 Fixed though!